src/Fmt_ast.ml

```diff
@@ -1518,7 +1517,7 @@ and fmt_expression c ?(box= true) ?epi ?eol ?parens ?ext ({ast= exp} as xexp)
   | Pexp_constraint (e, t) ->
       wrap_fits_breaks "(" ")"
         ( fmt_expression c (sub_exp ~ctx e)
-        $ fmt "@ : "
+        $ fmt "@;<1 2>: "
```

Comment:
I suspect that the whole body of the Pexp_constraint case should be wrapped in a box a la Ppat_constraint above, and `@;<1 2>` could go back to `@ `

Reply:
done